terraform/20-app/cloudwatch.canary.front-end.tf

```diff
@@ -0,0 +1,21 @@
+module "cloudwatch_canary_front_end_screenshots" {
+  source = "../modules/cloud-watch-canary"
+  name   = "${local.prefix}-canary-front-end"
+
+  create = false
```

Comment:
Update this to be:
```
create = local.is_prod
```
When the memory leak is fixed and the synthetic monitoring can safely go in